src/ducks/apps/components/ApplicationPage/index.jsx

```diff
@@ -9,6 +9,7 @@ import FocusTrap from 'focus-trap-react'
 import AppIcon from 'cozy-ui/react/AppIcon'
 
 import ApplicationPageLoading from 'ducks/components/ApplicationPageLoading'
+import { getTranslatedManifestProperty } from 'lib/helpers'
```

Comment:
Nit: I prefer namespaced functions

```
import manifest from 'lib/manifest'

manifest.getTranslatedProperty()
```

Reply:
Not a bad idea but I would keep like this for two reasons (at least for now), we use a lot `manifest` variable name which means the application manifest in the code and we only have one helper relative to the app manifest here.